### PR TITLE
lowers secway tankiness

### PR DESCRIPTION
**Pull request by ynot01**
# Document the changes in your pull request

300 -> 60

it blocks bullets so it basically quadruples your health

# Changelog

:cl:  
tweak: significantly lowered secway integrity
/:cl:

### CODE REVIEW

**Comment by Marmio64** [comment]
40 is a little bit low, a fellow officer could accidently miss 2 lasers and hit you and now your 5000 credit toy is utterly destroyed. It'd be fine if you made it so the secway is only temporarily destroyed and you can fix it up with a welder or something (maybe make it drop a wreckage when destroyed?), but otherwise 40 health is just too low. How about 100 health, or maybe 80?

**Comment by ynot01** [comment]
> 40 is a little bit low, a fellow officer could accidently miss 2 lasers and hit you and now your 5000 credit toy is utterly destroyed. It'd be fine if you made it so the secway is only temporarily destroyed and you can fix it up with a welder or something (maybe make it drop a wreckage when destroyed?), but otherwise 40 health is just too low. How about 100 health, or maybe 80?

40 hp still increases your effective health by 40, it is still strong, i will look into repairs though

**Comment by ilikesaying** [comment]
40 health with no armor, only able to survive like 3 hits

File: code/modules/vehicles/secway.dm [hunk 1]
```diff
@@ -4,6 +4,7 @@
 	desc = "A brave security cyborg gave its life to help you look like a complete tool."
 	icon_state = "secway"
 	key_type = /obj/item/key/security
+	max_integrity = 40
```
Comment (JamieD1) [inline code/modules/vehicles/secway.dm]:
```suggestion
	max_integrity = 60
```

**Review by JamieD1 — APPROVED** [review]

**Pull request merged**